ORBIT/manager.py

```diff
@@ -1141,7 +1194,12 @@ def phase_dates(self):
 
         for phase, _start in self.config["install_phases"].items():
 
-            start = dt.datetime.strptime(_start, self.date_format_short)
+            try:
+                start = dt.datetime.strptime(_start, self.date_format_short)
+
+            except TypeError:
+                start = self.start_date + dt.timedelta(hours=ceil(self.phase_starts[phase]))
```

Comment:
Could we move away from `ceil` for both `start` and `end` ? It won't result in well rounded off times, but should match the action logs exactly.

Reply:
I removed it here and in a few different places and ran the tests - everything still works. I had to keep it in the weather profile indexing but that shouldn't affect these outputs. It just internally rounds the time up to the nearest index location when getting a weather profile for an install phase.